power.py

```diff
@@ -9,7 +9,7 @@ def __init__(self):
         resMan = pyvisa.ResourceManager()
         resList = resMan.list_resources()
         try:
-            self._z60 = resMan.open_resource(resList[0])
+            self._z60 = resMan.open_resource(resList[4])
```

Comment:
Probably don't want to add this change since it won't apply to everyone's computer.